engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -223,6 +224,13 @@ class WorkflowActor(val workflowId: WorkflowId,
     case Event(WorkflowExecutionFailedResponse(executionStore, outputStore, failures),
     data @ WorkflowActorData(_, Some(workflowDescriptor), _, _)) =>
       finalizeWorkflow(data, workflowDescriptor, executionStore, outputStore, Option(failures.toList))
+    case Event(msg @ EngineStatsActor.JobCountQuery, data) =>
```

Comment:
FWIW that also causes it to lock up. 

I'm probably not going to have time to dive into 'why' prior to people wanting to release although we'll see how that plays out.

I don't personally care too much about the aborting thing right now